sdk/samples/azidentity/manual-tests/managed-identity/service-fabric/go.mod

```diff
@@ -0,0 +1,8 @@
+module main
+
+go 1.15
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.14.0
```

Comment:
This isn't the latest version, will that have any impact?

Reply:
Done, updated to latest.